charts/rstudio-connect/NEWS.md

```diff
@@ -1,3 +1,14 @@
+# 0.4.0
+
+- BREAKING: change `pod.nodeSelector` to `nodeSelector` for consistency with other charts
+  and the community. We error if this value is used to help highlight the change.
```

Comment:
```suggestion
  and the community. We error if this value is used to help highlight the change.
```

This sentence could use some clarification, not sure what we're trying to say/convey here